packages/components/pie-textarea/test/component/pie-textarea.spec.ts

```diff
@@ -654,7 +726,7 @@ test.describe('PieTextarea - Component tests', () => {
             expect(await page.evaluate(() => document.querySelector('pie-textarea')?.value)).toBe('');
         });
 
-        test('should correctly reset the textarea value to the `defaultValue` if one is provided when the form is reset', async ({ page }) => {
+        test('should correctly set the textarea value even if `defaultValue` is provided', async ({ page }) => {
```

Comment:
I don't think this test needs the textarea to be inside a form or be alongside a reset button, so we can probably just use `mount` like most of the other tests?